### DISCUSSION TITLE
Numpy methods

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#22] [Q&A] [answered]
flatten() is used for?

**Comment by Divk-Ashwin** [comment, marked as answer]
Flatten() is a method used in numpy library. It is used in arrays concept, its function is to convert 2D or 3D arrays into 1D array,
For example lets say, there is an array arr1=[[1,2],[3,4]] which is a 2D array when we use flatten() function, it converts arr1 into a 1D array which will give an output as arr_flattened=[1,2,3,4]

**Comment by 17-Manish** [comment]
flatten is used to take a list of lists or arrayand turn it into a single list or array
for eg,[[2,4,5]] the result will be [2,3,5]

**Comment by TRahulsingh** [comment]
flatten() simplifies multi-dimensional arrays into a single line of data, making it easier to work with.
